frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts

```diff
@@ -60,7 +60,8 @@ export function getPositionStatusOf(tabSetNode: TabSetNode): NodePositionStatus
   // of the root container that contains everything except for the borders.
   isLeftMost = tabSetNodeRect.x === rootContainerRect.x;
   isRightMost =
-    tabSetNodeRect.x + tabSetNodeRect.width === rootContainerRect.x + rootContainerRect.width;
+    tabSetNodeRect.x + tabSetNodeRect.width ===
+    Math.trunc(rootContainerRect.x + rootContainerRect.width);
```

Comment:
rootContainer.x and .width contain fractions in some case. This caused the comparisons to be, .e.g `25 == 25.234543`

Reply:
Would it be even more fail-safe to also truncate the `tabSetNodeRect` values in line 63? From the documentation I cannot see why `rootContainerRect` could sometimes be float but `tabSetNodeRect` could not.

Reply:
I have no idea how the fractions are derived. In my testing it worked with 80% - 250% browser scaling without issues.